test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -0,0 +1,215 @@
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.ImageAnalytics;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using System.Runtime.InteropServices;
+using Microsoft.ML.OnnxScoring;
+
+namespace Microsoft.ML.Tests
+{
+    public class OnnxTransformTests : TestDataPipeBase
+    {
+
+        private const int inputsize = 150528;
+
+        private class TestData
+        {
+            [VectorType(inputsize)]
+            public float[] data_0;
+        }
+        private class TestDataSize
+        {
+            [VectorType(2)]
+            public float[] data_0;
+        }
+        private class TestDataXY
+        {
+            [VectorType(inputsize)]
+            public float[] A;
+        }
+        private class TestDataDifferntType
+        {
+            [VectorType(inputsize)]
+            public string[] data_0;
+        }
+
+        private float[] getSampleArrayData()
+        {
+            var samplevector = new float[inputsize];
+            for (int i = 0; i < inputsize; i++)
+                samplevector[i] = (i / (inputsize * 1.01f));
+            return samplevector;
+        }
+
+        public OnnxTransformTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        void TestSimpleCase()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
>if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) [](start = 12, length = 54)

Is it working on MacOS?
If no set specific to IsOSPlatform(OSPlatform.Windows)

Reply:
Good point. Done.

---
In reply to: [220397906](https://github.com/dotnet/machinelearning/pull/942#discussion_r220397906) [](ancestors = 220397906)